### ISSUE TITLE
Convert Geogebra API JS into C# Code

### ISSUE THREAD

**Issue by nsgnn** [maine-imre/handwaver#64]
**Is your feature request related to a problem? Please describe.**

- This is related to issue #9.

**Describe the solution you'd like**

-  It is a subtask that would create callable code within C# from Geogebra JavaScript.

**Describe alternatives you've considered**

- JavaScript.Net
- IKVM

**Additional Info**
```
The desired Code to be callable within C#:
https://wiki.geogebra.org/en/Reference:GeoGebra_Apps_API
A suggested tool to call JS within C#:
https://archive.codeplex.com/?p=javascriptdotnet
A suggested tool to create a DLL that contains a jar file:
https://www.ikvm.net/
```

**Comment by nsgnn** [comment]
[Here](https://github.com/maine-imre/Javascript.Net/releases/tag/Feb2019Test) is a built dll for the Javascript.net code. With this, test within a unity project how to call functions from JS in the c# code. You can see reference docs [here.](https://github.com/JavascriptNet/Javascript.Net/wiki) I would start with a simple hello world script and see if you can get it debugged out into the unity console. Along the way, you will have some issues with some things you want to do in unity. This is where you will have to look up online how to achieve your desired behaviour within the code. If there is something you are stumped on feel free to message me about it on discord.

**Comment by nsgnn** [comment]
We have attempted many different things since last update. Managing geogebra code was not as easy as expected. After getting it close to compiling, we discovered a hardcoded dependency on the eclipse compiler. This is not nice as we are using IntelliJ IDEA. The move has been made to attempting use of Node-JS-Geogebra. This would either be server side or locally hosted. At scale, this might be an issue, but for now it handles what is needed of it.
